src/html/output-debug.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+function debug(payload, constants, logger) {
+  // TODO add debug query parameter check once available!
```

Comment:
w/o this check, this PR is not applicable.

Reply:
Yes, working on identifying who should be responsible of storing the query params in the payload.

Reply:
the problem is, that the params are merged with the default actions in openwhisk. so my making them available, we would also include those, which might contain secret variables. but I'm not sure.

Reply:
looking at https://github.com/adobe/openwhisk-loggly-wrapper/blob/master/wrap.js#L95, I think the params are already in `secrets`

Reply:
That's where I am now... This becomes really error prone. If I expect a `path` query param, it will conflict with the openwhisk action `path`.. Same for owner, ref and repo.
We should separate them.

Reply:
Created https://github.com/adobe/petridish/issues/46 to continue the discussion